src/components/home/home.jsx

```diff
@@ -80,6 +80,14 @@ class HomePage extends React.Component {
               </ul>
         </section>
 
+        <section id="home-volunteer">
+            <h2>Want to join the team?</h2>
+            <p>We welcome volunteers!</p>
+            <p>We work remotely and asynchronously using online collaboration tools like GitHub.<br />
+            Our stack is JavaScript/NodeJS, React, React-Native, and MongoDB.</p>
+            <button href="https://forms.gle/4ERxvyu5ThMC7dGU7" className="cta-btn">Volunteer</button>
```

Comment:
Clicking the button does not work. Buttons need to be surrounded by "a" tags. Href attribute is not valid for buttons but is valid for "a" tags. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button
https://www.w3schools.com/tags/tag_a.asp

Reply:
Well, that's embarrassing. Not only did I know that, I didn't test the button.

Now that I have tested, the provided link doesn't work. It looks like the form is protected?